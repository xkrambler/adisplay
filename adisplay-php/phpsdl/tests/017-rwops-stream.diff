002+ 
003+ Warning: SDL_RWFromFP(): supplied argument is not a valid stream resource in /data/dev/accord_display8th/adisplay/phpsdl/tests/017-rwops-stream.php on line 4
004+ size:
005+ Fatal error: Uncaught Error: Call to a member function size() on boolean in /data/dev/accord_display8th/adisplay/phpsdl/tests/017-rwops-stream.php:5
002- size:int(20932)
003- read:bool(true)
004- bool(true)
005- = Done
006+ Stack trace:
007+ #0 {main}
008+   thrown in /data/dev/accord_display8th/adisplay/phpsdl/tests/017-rwops-stream.php on line 5