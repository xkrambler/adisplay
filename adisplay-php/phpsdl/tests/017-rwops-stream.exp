= Read Stream
size:int(20932)
read:bool(true)
bool(true)
= Done