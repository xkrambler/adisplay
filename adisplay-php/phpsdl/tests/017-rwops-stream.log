
---- EXPECTED OUTPUT
= Read Stream
size:int(20932)
read:bool(true)
bool(true)
= Done
---- ACTUAL OUTPUT
= Read Stream

Warning: SDL_RWFromFP(): supplied argument is not a valid stream resource in /data/dev/accord_display8th/adisplay/phpsdl/tests/017-rwops-stream.php on line 4
size:
Fatal error: Uncaught Error: Call to a member function size() on boolean in /data/dev/accord_display8th/adisplay/phpsdl/tests/017-rwops-stream.php:5
Stack trace:
#0 {main}
  thrown in /data/dev/accord_display8th/adisplay/phpsdl/tests/017-rwops-stream.php on line 5
---- FAILED
