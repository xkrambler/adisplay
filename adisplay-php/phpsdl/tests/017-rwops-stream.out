= Read Stream

Warning: SDL_RWFromFP(): supplied argument is not a valid stream resource in /data/dev/accord_display8th/adisplay/phpsdl/tests/017-rwops-stream.php on line 4
size:
Fatal error: Uncaught Error: Call to a member function size() on boolean in /data/dev/accord_display8th/adisplay/phpsdl/tests/017-rwops-stream.php:5
Stack trace:
#0 {main}
  thrown in /data/dev/accord_display8th/adisplay/phpsdl/tests/017-rwops-stream.php on line 5